src/wp-content/themes/twentytwentyone/functions.php

```diff
@@ -635,7 +663,6 @@ function twentytwentyone_add_ie_class() {
 	</script>
 	<?php
 }
-add_action( 'wp_footer', 'twentytwentyone_add_ie_class' );
```

Comment:
Since skip-link-focus-fix was deprecated, it seems sensible to do the same for this script.

Reply:
See above, this is an unrelated change.